userguide/getting-started/quick_start_with_kaui.adoc

```diff
@@ -0,0 +1,189 @@
+= Quick Start with Kaui
+:imagesdir: C:\_My Documents\FlowWritingLLC\Projects\Kill Bill\Documentation\killbill-docs\userguide\assets\img\getting-started
+
+Kaui is the back-office user interface for Kill Bill. This guide walks you through several tasks in Kaui. You will become familiar with moving around the application, and you'll learn some basic Kill Bill concepts:  
+
+. <<step1, Create a tenant>>
+. <<step2, Set up a simple catalog and plans>>
+. <<step3, Create an account>> 
+. <<step4, Add a payment method to the account>>
+. <<step5, Set up a subscription for the account>>
+. <<step6, View the generated invoice>>
+
+[NOTE]
+*Note:* You can also achieve these same results of this tutorial with the API. In fact, if you are going to have a front-end interface (such as a website), the API calls are what you use to communicate with Kill Bill (as opposed to using Kaui). For more information, see https://docs.killbill.io/latest/getting_started.html#_api_client[API Client] in the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide.
+
```

Comment:
There is a formatting issue with https://docs.killbill.io/latest/getting_started.html#_api_client, I think the underscore character needs to be escaped.

Reply:
Thanks! This should be fixed.